h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -677,6 +719,27 @@ public void validate(GLM glm) {
           throw new IllegalArgumentException("Missing values handling should be specified as an instance of " + MissingValuesHandling.class.getName());
         }
       }
+      if(_control_variables != null) {
+        if ((_interactions != null || _interaction_pairs != null)) {
+          glm.error("_control_variables", "Control variables option is not supported with interactions.");
+        }
+        for(String col: _control_variables){
+          Vec v = train().vec(col);
+          if (v == null) {
+            glm.error("_control_variables", "Control variable '"+col+"' is not in the frame.");
+          } else {
+            if (col.equals(_weights_column)){
+              glm.error("_control_variables", "Control variable '"+col+"' is set as _weight_column.");
```

Comment:
I think end users don't care for the internal naming convention so I would skip the underscore prefix in errors
```suggestion
              glm.error("_control_variables", "Control variable '"+col+"' is set as weight_column.");
```